receiver/kubeletstatsreceiver/internal/kubelet/metrics_test.go

```diff
@@ -88,6 +88,7 @@ func requireMetricOk(t *testing.T, m pmetric.Metric) {
 			require.Less(t, dp.StartTimestamp(), dp.Timestamp())
 			requirePointOk(t, dp)
 		}
+	case pmetric.MetricTypeEmpty:
```

Comment:
Looks like exhaustive lint requires the case to have a func body: https://github.com/open-telemetry/opentelemetry-collector-contrib/actions/runs/5403206833/jobs/9815696895?pr=23815#step:8:934